src/seaport.ts

```diff
@@ -1029,24 +1031,38 @@ export class Seaport {
     ]);
 
     const ordersMetadata: FulfillOrdersMetadata = fulfillOrderDetails.map(
-      (orderDetails, index) => ({
-        order: orderDetails.order,
-        unitsToFill: orderDetails.unitsToFill,
-        orderStatus: scaleOrderStatusToMaxUnits(
-          orderDetails.order,
-          orderStatuses[index],
-        ),
-        offerCriteria: orderDetails.offerCriteria ?? [],
-        considerationCriteria: orderDetails.considerationCriteria ?? [],
-        tips:
-          orderDetails.tips?.map((tip) => ({
-            ...mapInputItemToOfferItem(tip),
-            recipient: tip.recipient,
-          })) ?? [],
-        extraData: orderDetails.extraData ?? "0x",
-        offererBalancesAndApprovals: offerersBalancesAndApprovals[index],
-        offererOperator: allOffererOperators[index],
-      }),
+      (orderDetails, index) => {
+        const o = {
```

Comment:
could you use "order" instead of "o"?

Reply:
[931a81d](https://github.com/ProjectOpenSea/seaport-js/pull/559/commits/931a81d5e9bf4f8223efa891825fc60c236c3ac6)